ERCS/erc-321.md

```diff
@@ -0,0 +1,110 @@
+---
+eip: 321
```

Comment:
```suggestion
eip: 7625
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.